client/src/App.js

```diff
@@ -1,43 +1,11 @@
 import './App.css';
 import AppRouter from './navigation/AppRouter';
-import { createTheme, ThemeProvider } from '@material-ui/core';
+import { ThemeProvider } from '@material-ui/core';
```

Comment:
Is it possible to just 
`import ThemeProvider from '@material-ui/core/ThemeProvider'; instead of destructuring?

Reply:
you can do {ThemeProvider} from '@material-ui/core/styles'.